datasource/src/main/scala/quasar/physical/s3/S3DatasourceModule.scala

```diff
@@ -41,11 +44,13 @@ import slamdata.Predef.{Stream => _, _}
 object S3DatasourceModule extends LightweightDatasourceModule {
   def kind: DatasourceType = s3.datasourceKind
 
-  def lightweightDatasource[F[_]: ConcurrentEffect: MonadResourceErr: Timer](config: Json)
+  def lightweightDatasource[F[_]: ConcurrentEffect: ContextShift: MonadResourceErr: Timer](config: Json)
       : F[InitializationError[Json] \/ Disposable[F, Datasource[F, Stream[F, ?], ResourcePath]]] = {
+    val ec = ExecutionContext.Implicits.global
```

Comment:
The latest signature over in slamdata/quasar#3874 includes the implicit context.

Reply:
@jsantos17 in case you missed my comment on the quasar PR, you should be able to use the last published snapshot of that PR: 78.1.1-f21a31d